tests/functional/reset_password.js

```diff
@@ -617,20 +589,17 @@ define([
       return this.remote
         .get(require.toUrl(url))
         .setFindTimeout(intern.config.pageLoadTimeout)
-        .findByCssSelector('form input.email')
-          .getAttribute('value')
-          .then(function (resultText) {
-            // email address should be pre-filled from the query param.
-            assert.equal(resultText, email);
-          })
-        .end()
+
+        // email address should not be pre-filled from the query param.
+        .then(testElementValueEquals('form input.email'), '')
```

Comment:
This is the failing test, and it's only failing because the `''` should be before the first closing `)`:

```js
.then(testElementValueEquals('form input.email', ''))
```